framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -195,9 +196,34 @@ private void viewpointAdaptExecutable(
             receiver = FlowExpressions.internalReprOf(factory, receiverTree);
         }
 
-        List<FlowExpressions.Receiver> argReceivers = new ArrayList<>(args.size());
-        for (ExpressionTree argTree : args) {
-            argReceivers.add(FlowExpressions.internalReprOf(factory, argTree));
+        List<FlowExpressions.Receiver> argReceivers = new ArrayList<>();
+        boolean varargs = false;
+        if (tree instanceof MethodInvocationTree) {
+            ExpressionTree methodElement = ((MethodInvocationTree) tree).getMethodSelect();
+            ExecutableElement methodCalled =
+                    (ExecutableElement) TreeUtils.elementFromUse(methodElement);
+            if (methodCalled.isVarArgs()) {
```

Comment:
From the `isVarArgs` Javadoc 
> Returns true if this method or constructor accepts a variable number of arguments 

But that doesn't mean that a particular invocation passes variable arguments.  It could pass an array.